services/net/notification/Models/TnoRazorLightEngine.cs

```diff
@@ -0,0 +1,19 @@
+﻿using RazorLight;
+
+namespace TNO.Services.Notification.Models
+{
+    public class TnoRazorLightEngine : ITnoRazorLightEngine
```

Comment:
Not a model, place code in root of project.

Reply:
Updated, thanks